uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/AbsenceType.php

```diff
@@ -59,7 +58,7 @@ public function delete($absenceTypeID) {
     $absenceType = AbsenceType::findById($absenceTypeID);
 
     if ($this->absenceTypeHasEverBeenUsed($absenceType->id)) {
-      throw new UnexpectedValueException('Absence type cannot be deleted because it is linked to one or more leave requests');
+      throw new OperationNotAllowedException('Absence type cannot be deleted because it is linked to one or more leave requests');
```

Comment:
Now that this method doesn't throw an UnexpectedValueException anymore, you need to remove it from the docblock as well